### DISCUSSION TITLE
Why recommend RunnerUITests(UI Testing Bundle) but not RunnerTests(Unit Testing Bundle)

### DISCUSSION THREAD

**Discussion by ostk0069** [leancodepl/patrol#1063] [Q&A] [answered]
I just wondered why recomend RunnerUITests(UI Testing Bundle) but not RunnerTests(Unit Testing Bundle) in https://patrol.leancode.co/native/setup in iOS?
Because https://github.com/flutter/flutter/tree/main/packages/integration_test#ios-device-testing recommend to use `Unit Testing Bundle`
Is there any problems using Unit Testing Bundle with Patrol?

**Comment by bartekpacia** [comment, marked as answer]
Yes, Unit Testing Bundle works very differently to UI Testing Bundle. Patrol requires UI Testing Bundle to enable its native automation features.

That's also why we recommend naming the target `RunnerUITests` instead of `RunnerTests`, to make the distinction clear.

**Reply by ostk0069** [reply]
Thanks! I got it! 🙆‍♂️
